gui/default/syncthing/core/syncthingController.js

```diff
@@ -62,8 +62,9 @@ angular.module('syncthing.core')
         $scope.folderDefaults = {
             selectedDevices: {},
             type: "readwrite",
-            rescanIntervalS: 60,
+            rescanIntervalS: 3600,
             fsWatcherDelayS: 10,
+            fsWatcherEnabled: true,
```

Comment:
You need an equivalent change in go to have this enabled for introduced folders etc which does not involve js

Reply:
done